internal/providers/terraform/azure/automation_account.go

```diff
@@ -0,0 +1,75 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMAutomationAccountRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_automation_account",
+		RFunc: NewAzureRMAutomationAccount,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+	}
+}
+
+func NewAzureRMAutomationAccount(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyJobRunMins, nonAzureConfigNodeCount, monthlyWatcherHours *decimal.Decimal
+	location := lookupRegion(d, []string{"resource_group_name"})
```

Comment:
```suggestion
	location := lookupRegion(d, []string{})
```


This is not needed as lookupRegion automatically looks for this.

Reply:
@alikhajeh1 Done!